### DISCUSSION TITLE
weekly/2024-07-14

### DISCUSSION THREAD

**Discussion by giscus** [ATQQ/sugar-blog#267] [Announcements]
# weekly/2024-07-14

① YakShaving - 薅牛毛 ② CSS OneLiners ③ Vue Vine - 单文件编写多 Vue 组件 ④ CDN 流量被盗刷经历 ⑤ es-toolkit ⑥ console.log 体验优化 ⑦ 诗境 - 根据图片匹配诗句

https://sugarat.top/weekly/2024-07-14.html

**Comment by XiaofengdiZhu** [comment]
诗境的链接需要把最后的`/detail`去掉，否则打不开

**Reply by ATQQ** [reply]
好滴，我下来改下
